source/_data/glossary.yml

```diff
@@ -53,7 +53,7 @@
 - topic: Packages
   description: "[Packages](/docs/configuration/packages/) allow you to bundle different component configurations together."
 - topic: Platform
-  description: "[Platforms](/docs/configuration/platform_options/) make the connection to a specific software or hardware platform. For example, the `pushbullet` platform works with the service pushbullet.com to send notifications."
+  description: "[Platforms](/docs/configuration/platform_options/) is just a different name for integrations that isn't used that often anymore."
```

Comment:
This change is not correct. Platforms are still a vital part of Home Assistant. Some examples include: `light`, `switch`, `fan`, and all the other entities you see in general.

Not having the correct wording right now, but a platform basically allows another integration to use another integration domain.

For example, we have the [light integration](/integrations/light), which has all the services and basic building blocks. The [Hue](/integrations/hue) next uses the entity platform exposed by the light domain to provide lights to Home Assistant.

We also have platforms that do not expose entities, for example, the recently introduced `diagnostics` platform.

Anyways, this is probably the hardest one to explain and I wonder how far/deep explanation is needed even (as its more in general an implementation detail).

Reply:
Are there non-core platforms like a `hue` platform?
Because if not then maybe platforms are what I initially suspected domains were.

Reply:
Basically, platforms are integrations that provide building blocks other integrations can use. For example, the light integration provides the light platform that could be implemented by for example the Hue integration.

> Are there non-core platforms like a hue platform?

In theory, yes. We don't allow for them in core (as in we have a fixed set, like the "known" entity platforms, but a bunch of additional ones as well).

However, custom integrations could implement new ones (which is out of our control of course).

Reply:
> Basically, platforms are integrations that provide building blocks other integrations can use. For example, the light integration provides the light platform that could be implemented by for example the Hue integration.

That's pretty close to a definition :)

I've tried to use that for the latest edit and hope that I got it right this time.
 Thank you for taking your time to explain! I've learned a lot about the architecture today